jetpack/src/main/java/com/example/jetpack/viewbinding/base3/FragmentViewBinding.kt

```diff
@@ -53,38 +56,3 @@ class FragmentViewBindingDelegate1<VB : ViewBinding>(
         return viewBinding!!
     }
 }
-
-class FragmentViewBindingDelegate2<VB : ViewBinding>(
-    clazz: Class<VB>,
-) : ReadOnlyProperty<Fragment, VB> {
-    private var viewBinding: VB? = null
-
-    private val bindMethod = clazz.getMethod("bind", View::class.java)
-
-    private val mainHandler = Handler(Looper.getMainLooper())
-
-    override fun getValue(thisRef: Fragment, property: KProperty<*>): VB {
-        viewBinding?.let { return it }
-
-        val lifecycle = thisRef.viewLifecycleOwner.lifecycle
-        viewBinding = bindMethod.invoke(null, thisRef.requireView()) as VB
-        if (lifecycle.currentState == Lifecycle.State.DESTROYED) {
-            Log.w(
-                "TAG",
-                "Access to viewBinding after Lifecycle is destroyed or hasn't created yet. The instance of viewBinding will be not cached."
```

Comment:
Also: notice that this is not reflecting what's happening. `this.viewBinding` was just assigned in the previous line, therefore caching it.

Reply:
I have a feeling this will never happen. `requireView` will throw an exception before this problem can be detected.